src/pages-and-resources/discussions/app-list/messages.js

```diff
@@ -0,0 +1,74 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  heading: {
+    id: 'authoring.discussions.heading',
+    defaultMessage: 'Select a discussion tool for this course',
```

Comment:
I saw a PR by @asadazam93 who fixed this string I think you would need to port that in? or rebase once that's merged? 
https://github.com/edx/frontend-app-course-authoring/pull/72#issuecomment-814917529

Reply:
I can fix it once that's merged, yeah.  Thanks for the heads up!

Reply:
I see that PR merged - will rebase on master now